.buildkite/pipeline-release-helm.yml

```
@@ -0,0 +1,41 @@
+steps:
+  - group: helm-release
+    steps:
+
+      - label: "operator dev helm chart"
+        if: |
+          ( build.branch == "main" && build.source != "schedule" )
+          || build.tag != null
+          || build.message =~ /^buildkite test .*release.*/
+          || build.message == "release eck-operator helm charts"
+        key: "operator-dev-helm"
+        commands:
+          - make -C hack/helm/release build 2> /dev/null
+          - hack/helm/release/bin/releaser --env=dev --charts-dir=deploy/eck-operator
+
+      - label: "stack dev helm charts"
```

Maybe to be consistent with naming?

```suggestion
      - label: "eck-resources dev helm charts"
```